shared/uavobjectdefinition/lqgsettings.xml

```diff
@@ -8,7 +8,7 @@
     <field defaultvalue="25, 75, 1, 0.000003, 0.000001, 0.3" name="RTKF" type="float" units="" elementnames="R, YawR, Q1, Q2, Q3, BiasLimit">
         <description>Weights for the Rate-Torque Kalman filter.</description>
     </field>
-    <field defaultvalue="0.000025, 0.0001, 0, 0.000025, 0.0001" name="LQR" type="float" units="" elementnames="Q1, Q2, YawBetaOffset, YawQ1, YawQ2">
+    <field defaultvalue="1, 0.000025, 0.0001, 1, 0.000025, 0.0001" name="LQR" type="float" units="" elementnames="R, Q1, Q2, YawR, YawQ1, YawQ2">
```

Comment:
I'm concerned about import/export of this, that it will shuffle fields.  e.g. dumps look like

```
      <field name="GyroVar" values="9.99999974738e-06,9.99999974738e-06,9.99999974738e
-05"/>
```

Maybe rename the field?

Reply:
The change in element count (5 to 6) doesn't break import? If so, I can rename it to LQRegulator I guess.

Reply:
Nope it doesn't. Renaming.